README.md
Elenis
======


history.txt
    1  mkdir /home/i95dev
    2  cd ~
    3  ls
    4  cd ..
    5  ls
    6  cd /home
    7  ls
    8  cd i95dev/
    9  ls -la
   10  nano /etc/bash.bashrc 
   11  exit
   12  passwd
   13  useradd i95dev
   14  passwd i95dev
   15  cd /home
   16  ls
   17  cd ..
   18  usermod -a -G sudo i95dev
   19  exit
   20  ls
   21  cd ..
   22  cat /var/log/dmesg
   23  cat /var/log/faillog 
   24  cat /var/log/auth.log 
   25  apt-get install lamp-server^
   26  apt-get update
   27  apt-get upgrade 
   28  apt-get install lamp-server^
   29  apt-get install php5-gd
   30  apt-get install php5-dom
   31  apt-get install php5-iconv
   32  apt-cache search iconv
   33  apt-get install perl libnet-ssleay-perl openssl libauthen-pam-perl libpam-runtime libio-pty-perl apt-show-versions python
   34  nano /etc/apt/sources.list
   35  apt-get update
   36  cd ~
   37  wget http://www.webmin.com/jcameron-key.asc
   38  apt-key add jcameron-key.asc
   39  apt-get update
   40  apt-get upgrade
   41  apt-get install webmin
   42  php -m
   43  apt-cache search mcrypt
   44  apt-get install php5-mcrypt
   45  php -m
   46  apt-get install php5-curl
   47  apt-get install php5-soap
   48  apt-cache search php soap
   49  apt-get install php-soap
   50  apt-get install postfix
   51  apt-get install phpmyadmin
   52  nano /etc/apache2/apache2.conf
   53  service apache2 restart
   54  nano /etc/phpmyadmin/apache.conf
   55  nano /etc/apache2/apache2.conf
   56  service apache2 restart
   57  nano /etc/phpmyadmin/apache.conf
   58  nano /etc/apache2/apache2.conf
   59  nano /etc/phpmyadmin/apache.conf
   60  nano /usr/share/phpmyadmin/.htaccess
   61  a2enmod ssl
   62  service apache2 restart
   63  nano /usr/share/phpmyadmin/.htaccess
   64  htpasswd -c /etc/phpmyadmin/.htpasswd i95dev
   65  service apache2 restart
   66  exit
   67  ls
   68  df -h
   69  cd /home/
   70  ls
   71  cd /var/www/
   72  ls
   73  cd ../
   74  ls
   75  ifconfig 
   76  mysql -udbadmin1 -p
   77  php -v
   78  free -m
   79  cd /etc/apache2/sites-available/
   80  ls
   81  ls -lah
   82  cd ../sites-enabled/
   83  ls -lah
   84  vi 000-default 
   85  ls
   86  df -h
   87  php -v
   88  ifconfig 
   89  ls
   90  wget http://dev.i95dev.com/elenis_main_21092013.tgz
   91  ls
   92  rm -rf elenis_main_21092013.tgz 
   93  nohup wget http://dev.i95dev.com/elenis_main_21092013.tgz &
   94  ls
   95  du -sh elenis_main_21092013.tgz 
   96  cd /var/www/
   97  ls
   98  vi index.php
   99  history
  100  apt-get install php5-xmlrpc
  101  apache2ctl restart
  102  vi /etc/phpmyadmin/apache.conf 
  103  cd 
  104  ls
  105  du -sh elenis_main_21092013.tgz 
  106  ls
  107  tar -xvzf elenis_main_21092013.tgz 
  108  ls
  109  mv elenis_main elenidev2.vanwestmedia.com
  110  mv elenidev2.vanwestmedia.com /var/www/
  111  ls
  112  cd /var/www/
  113  ls
  114  cd elenidev2.vanwestmedia.com
  115  ls
  116  cd app/etc/
  117  cp local.xml local.xml_bak
  118  vi local
  119  vi local.xml
  120  ls
  121  cd ../
  122  cd var/cache/
  123  ls
  124  rm -rf *
  125  cd ../full_page_cache/
  126  ls
  127  rm -rf *
  128  cd ../
  129  ls
  130  mysql -udbadmin1 -p895e3f!
  131  mysql -udbadmin1 -p895e3f! elenidev2_vanwestmedia_com < elenis_main_21092013.sql 
  132  mysql -udbadmin1 -p895e3f!
  133  cd /etc/apache2/sites-available/
  134  ls
  135  mysql -udbadmin1 -p895e3f!
  136  ls
  137  cp default elenidev2.vanwestmedia.com
  138  ls
  139  vi elenidev2.vanwestmedia.com 
  140  ls -lah
  141  ln -s /etc/apache2/sites-available/elenidev2.vanwestmedia.com /etc/apache2/sites-enabled/elenidev2.vanwestmedia.com
  142  cd ../sites-enabled/
  143  ls
  144  ls -lah
  145  apache2ctl restart
  146  ls
  147  cd ../
  148  cd /var/www/
  149  ls
  150  rm -rf index.html index.php 
  151  ls
  152  cd ../
  153  ls
  154  ls -lah
  155  mv www www-old
  156  cd ../
  157  cd /var/www-old/
  158  ls
  159  mv elenidev2.vanwestmedia.com www
  160  ls
  161  mv www ../
  162  ls
  163  cd ../
  164  ls -lah
  165  chown -R root:root www
  166  ls -lah
  167  chmod -R 755 www
  168  cd www
  169  chmod -R 777 app/etc media var/
  170  ls -lah
  171  cd ../
  172  cd /etc/apache2/sites-available/
  173  ls
  174  ls -lah
  175  cd ../sites-enabled/
  176  ls
  177  rm -rf elenidev2.vanwestmedia.com 
  178  ls
  179  cd ../
  180  cd sites-available/
  181  ls
  182  rm -rf elenidev2.vanwestmedia.com 
  183  ls
  184  cd /var/www/
  185  ls
  186  cd /var/www
  187  vi robots.txt 
  188  vi robots.txt
  189  cd
  190  cd /var/
  191  ls
  192  ls -lah
  193  vi .htaccess
  194  cd www
  195  vi .htaccess
  196  htpasswd -c /opt/.htpasswd
  197  htpasswd -c /opt/.htpasswd elenidev2
  198  vi .htaccess
  199  apache2ctl restart
  200  vi /etc/apache2/sites-available/default
  201  cd /etc/apache2/sites-available/
  202  cp default default_bak
  203  vi default
  204  apache2ctl restart
  205  vi default
  206  apache2ctl restart
  207  vi default
  208  apache2ctl restart
  209  ls
  210  pwd
  211  cd /home/
  212  ls
  213  cd i95dev/
  214  ls
  215  ls -lah
  216  cd ../
  217  ls -lah
  218  chown -R i95dev:i95dev i95dev/
  219  vi /etc/passwd
  220  cd i95dev/
  221  ls
  222  tar -xvzf elenis_main_07102013.sql.tgz 
  223  ls
  224  history|grep mysql
  225  mysql -udbadmin1 -p895e3f!
  226  mysql -udbadmin1 -p895e3f! elenidev2_vanwestmedia_com_07102013 < elenis_main_07102013.sql
  227  passwd
  228  exit
  229  df -h
  230  ls
  231  watch ls -lah
  232  watch ls -lah | grep images
  233  watch ls -lah | grep image
  234  watch ls -lah images
  235  watch ls -lah
  236  ls
  237  ls -lah
  238  wget http://elenis.com/site_media/elenisdbbackupgavin.sql.tar.gz
  239  ls -lah
  240  dig johnapel.com NS
  241  dig johnapel.com A
  242  dig www.johnapel.com A
  243  exit
  244  mysql -udbadmin1 -p895e3f!
  245  pwd
  246  cd /home/i95dev/
  247  mysql -udbadmin1 -p895e3f! elenidev2_vanwestmedia_com_10_08_13 < elenis_main_08_10_13.sql 

orderExport1.php
<?php

//echo "123";

// include the core code we are going to use
require_once('app/Mage.php');
umask (0);
Mage::app('default');
//$orders = array(0=>'139');
 
 $order_id=182;
 $order = Mage::getModel('sales/order')->load($order_id);
  
  echo "<pre>";
   $orderCreatedDate = '12/27/2013  2:38:43 PM';
   $orderCreatedDate1 =  explode(' ',$orderCreatedDate);
    
   
   
   
   
  // print_r($orderCreatedDate1[0]);
  //  echo $orderCreatedDate1['0'];
    $orderData= $order->getData();
  $payarry=$order->getPayment()->getData();
 
 
   $cc_exp_month = $payarry['cc_exp_month'] ? $payarry['cc_exp_month'] : '';
  $cc_exp_year = $payarry['cc_exp_year'] ? $payarry['cc_exp_year'] : '';
 
 $ccExp = $cc_exp_month.'/'.'01'.'/'.$cc_exp_year;
 $expiryDdate =  (string)date("m/Y", strtotime($ccExp)); 
 
 $deliveryDate =  $order->getDeliveryDate();
     
        if($deliveryDate!=''){
         $deliveryDate = date("m/d/Y", strtotime($deliveryDate));
        }
 
  print_r($orderData);exit;
       
  $sagepaymentspro = Mage::getModel('ebizmarts_sagepaymentspro/transaction')->getCollection()
                        ->addFieldToFilter('order_id', $order_id);
        
         $postCodeReusult='';
        foreach($sagepaymentspro as $transaction) {
          
            $postCodeReusult= $transaction->getPostCodeResult();
             
        }
  
  // $postCodeReusult;
   //  $shippingAddress = !$order->getIsVirtual() ? $order->getShippingAddress() : null;
    
  //   print_r($shippingAddress->getData());
   //  echo $this->getShippingMethod($order);
   
          die('121');  
           
       
 
?>
